components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/dto/TierDTO.java

```diff
@@ -23,6 +23,12 @@
   
   private String description = null;
   
+  public enum TierLevelEnum {
+     api,  application,  resource, 
```

Comment:
Yes this is used for a path parameter in the GET /tiers URI in REST API GET /tiers/{tier-level}. This is for the {tier-level} parameter.